lib/matplotlib/axes/_axes.py

```diff
@@ -36,6 +36,25 @@
 from matplotlib.axes._base import _AxesBase, _process_plot_format
 from matplotlib.axes._secondary_axes import SecondaryAxis
 
+try:
+    from numpy.lib.histograms import histogram_bin_edges
+except ImportError:
+    def histogram_bin_edges(arr, bins, range=None, weights=None):
```

Comment:
add a comment mentioning which version of numpy this was introduced in (so that we can remove the backport when possible)